CHANGELOG-PENDING.md

```diff
@@ -17,6 +17,11 @@ Month, DD, YYYY
 
 - [ci] [Add more linters #219](https://github.com/celestiaorg/optimint/pull/219) [@tzdybal](https://github.com/tzdybal/)
 - [deps] [Update dependencies: grpc, cors, cobra, viper, tm-db #245](https://github.com/celestiaorg/optimint/pull/245)  [@tzdybal](https://github.com/tzdybal/)
+- [rpc] [Implement NumUnconfirmedTxs #255](https://github.com/celestiaorg/optimint/pull/255)  [@tzdybal](https://github.com/tzdybal/)
+- [rpc] [Implement Commit method #236](https://github.com/celestiaorg/optimint/pull/258)  [@raneet10](https://github.com
```

Comment:
Please add only one line for this entire PR (here we're linking per PR, not per issue)